python/rikai/__version__.py

```diff
@@ -11,3 +11,5 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+
+version = "0.0.1.dev0"
```

Comment:
this makes import a little bit non nature?
as it need `rikai.__version__.version` to get the string

Reply:
i'll import it from __init__ so you can do `rikai.version`

Reply:
Cool, feel free to merge after this